h/static/scripts/drafts.js

```diff
@@ -0,0 +1,84 @@
+/**
+ * The drafts service provides temporary storage for unsaved edits
+ * to new or existing annotations.
+ *
+ * A draft consists of a 'model' which is the original annotation
+ * which the draft is associated with and `changes' which is
+ * a set of edits to the original annotation.
+ */
+function DraftStore() {
```

Comment:
The name of the service is inconsistent with the name of the module. Perhaps rename `drafts` to `draft-store`?